src/commands/emasser/delete/poams.ts

```diff
@@ -9,7 +9,7 @@ import {outputError} from '../../../utils/emasser/outputError'
 import {FlagOptions, getFlagsForEndpoint} from '../../../utils/emasser/utilities'
 
 export default class EmasserDeletePoams extends Command {
-  static usage = '<%= command.id %> [ARGUMENTS]';
+  static usage = '<%= command.id %> [options]';
```

Comment:
What is the difference between ARGUMENTS and options?

Reply:
options are actually the FLAGS, I contemplated what to put there.
I think I should just change all to [FLAGS] - what do you think?